#! /bin/sh /usr/share/dpatch/dpatch-run
## 65_handle_templates_directories.dpatch by  <tfheen@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Skip directories when updating

@DPATCH@
diff -urNad mailman-2.1.5/bin/update /tmp/dpep.3rh2pG/mailman-2.1.5/bin/update
--- mailman-2.1.5/bin/update	2005-01-10 16:21:05.917239616 +0100
+++ /tmp/dpep.3rh2pG/mailman-2.1.5/bin/update	2005-01-10 16:21:56.833499168 +0100
@@ -127,7 +127,7 @@
         try:
             fp = open(os.path.join(mm_cfg.TEMPLATE_DIR, gtemplate))
         except IOError, e:
-            if e.errno <> errno.ENOENT: raise
+            if not (e.errno in [errno.ENOENT, errno.EISDIR, ]): raise
             # No global template
             continue
 
